### PR TITLE
APP-1662: Support variable logos for contract cards

### PR DESCRIPTION
**Pull request by oscarnylander**
Draft until the backend is complete (currently errors on many types of contracts)

### CODE REVIEW

**Review by hugokallstrom — APPROVED** [review]
LGTM!

File: app/src/main/res/layout/insurance_contract_card.xml [hunk 1]
```diff
@@ -69,12 +69,13 @@
...
 
                 <ImageView
+                    android:id="@+id/logo"
                     android:layout_width="@dimen/contract_card_logo_size"
                     android:layout_height="@dimen/contract_card_logo_size"
                     android:contentDescription="@null"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintTop_toTopOf="parent"
-                    app:srcCompat="@drawable/ic_hedvig_h"
+                    tools:src="@drawable/ic_hedvig_h"
```
Comment (StylianosGakis) [inline app/src/main/res/layout/insurance_contract_card.xml:78]: Is there any specific reason for this change? I don't know from the top of my head what the difference is, but usually "compat = good" 😂

**Review by StylianosGakis — APPROVED** [review]

File: app/src/main/res/layout/insurance_contract_card.xml (hunk 1 above)
Comment (oscarnylander) [reply app/src/main/res/layout/insurance_contract_card.xml:78]: The `ImageView` no longer loads an image resource from disk, but instead gets its image from `coil`. The switch to `tools:src` was merely to make sure something is displayed in the preview window. 🙂

Comment (StylianosGakis) [reply app/src/main/res/layout/insurance_contract_card.xml:78]: Oh "tools"! Missed the most important change here duh

**Pull request merged**